aten/src/THC/generic/THCTensor.cpp

```diff
@@ -237,7 +210,7 @@ THCTensor *THCTensor_(newFoldBatchDim)(THCState *state, THCTensor *input) {
   for (int i = 2; i < in_dims; i++) {
     new_size[i - 1] = THCTensor_(size)(state, input, i);
   }
-  THCTensor *output = THCTensor_(newView)(state, input, new_size);
+  THCTensor *output = at::native::view(THTensor_wrap(input), new_size).unsafeGetTensorImpl();
```

Comment:
@gchanan is this OK?

Reply:
Looks ok to me, but I'm not a TH expert

Reply:
lgtm, you can look at the generated code for this boundary (which calls checked_tensor_unwrap) and it does the same thing